sass/components/_site-user-menu.scss

```diff
@@ -24,11 +24,12 @@ a.site-user-menu__list {
 
 .site-user-menu__list {
   @extend .reset-lists;
+
   font-weight: $font-weight-bold;
   padding: 0.5em;
 
   .site-user-menu--header & {
-    background: #ececec;
+    background-color: #ececec;
```

Comment:
Hi @gingapa 
If you see my notes, that's is one of my listed issues in the comments of the PR.
It can ec-grey but only Filipa can confirm.